docs/content/en/index.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
I left some general comments from my side everywhere, just have a look at my comments.  
I am currently swamped in my dayjob and don't want to touch source code at home at the moment.

I would appreciate if you had some feedback about my other comments, but I don't mind if you go through with whatever you like.
I hope I have time to reply, if need be remove me from the review process - my schedule might return to normal around mid-november.

Thanks for the documentation do-over, I always had a feeling the readme.md was getting pretty long : )  
It's very much appreciated!